editor/components/block-list/block.js

```diff
@@ -647,6 +653,9 @@ const applyWithSelect = withSelect( ( select, { uid, rootUID } ) => {
 		block,
 		isSelected,
 		hasFixedToolbar,
+		rootUIDOfRoot: getBlockRootUID( rootUID ),
+		orderOfRoot: getBlockIndex( rootUID, getBlockRootUID( rootUID ) ),
+		isSelected,
```

Comment:
Duplicated (already assigned above). Thinking there ought to be an ESLint rule for this.

_Edit:_ [There is](https://eslint.org/docs/rules/no-dupe-keys) and it's enabled. I'm guessing it doesn't support the ES2015 property shorthand in the version we're using.

Reply:
>Duplicated (already assigned above).

Removed in 11d7161d16b97b3b0486385bbc46c123586f889d.